catalog-info.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: backstage.io/v1alpha1
+# This is the name of the component that will show up in the sidebar
+kind: Component
+metadata:
+  name: KotlinAudio
+  title: kotlinaudio
+  description: The repo that supports the submodule which is a fork of the original KotlinAudio but allows us to use reflection to integrate Conviva
+  annotations:
+    backstage.io/techdocs-ref: dir:.
```

Comment:
I would add a docs folder with an index.md file otherwise I think the techdocs page will be broken in backstage

Reply:
Can you double check the path @Matilde90